src/frontend/screens/Game/GamePage/components/ReleaseDate.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useContext } from 'react'
+import ContextProvider from 'frontend/state/ContextProvider'
+import { useTranslation } from 'react-i18next'
+
+type ReleaseDateProps = {
+  date: string[] | undefined
+  isLinuxNative: boolean
+  isMacNative: boolean
+}
+
+const ReleaseDate: React.FC<ReleaseDateProps> = ({
+  date,
+  isLinuxNative,
+  isMacNative
```

Comment:
we should be able to get these 2 and `isLinux` and `isMac` all from the GameContext so we don't need to pass them as props and we don't need to check `platform === ...`

```ts
const { is } = useContext(GameContext)
```

and then

```ts
      if (
        (platformName === 'Linux' && is.linuxNative && is.linux) ||
        (platformName === 'OS X' && is.macNative && is.mac)
      ) {
        return releaseDate
      }
```

Reply:
I tottally forgot about this new context you added on your refactor. Nice!